library/include/ck/library/reference_tensor_operation/cpu/reference_conv_fwd.hpp

```diff
@@ -57,10 +58,12 @@ struct ReferenceConvFwd : public device::BaseOperator
                  std::vector<ck::index_t> input_right_pads,
                  InElementwiseOperation in_element_op,
                  WeiElementwiseOperation wei_element_op,
-                 OutElementwiseOperation out_element_op)
+                 OutElementwiseOperation out_element_op,
+                 const std::array<Tensor<OutDataType>, NumDTensor>& postop_tensors)
```

Comment:
I'm not sure it makes sense to introduce a new nomenclature here. Do we use the name `postop` anywhere else? I guess it could just be called `d_tensors` and it would be pretty consistent and less confusing?